src/txdb.h

```diff
@@ -20,7 +20,7 @@ struct CDiskTxPos;
 class uint256;
 
 //! -dbcache default (MiB)
-static const int64_t nDefaultDbCache = 100;
+static const int64_t nDefaultDbCache = 700;
```

Comment:
I'm worried about this default on Windows. It's currently running in 32bit and using 1.2GB of RAM. This will bump it up against the 2GB barrier on Windows.

Reply:
So 450 or 500 as Kay suggest?

Reply:
What about `sizeof(void*) >= 8 ? 700 : 400`.

Reply:
That's good.